sticker-graph/sticker_graph/model.py

```diff
@@ -143,27 +159,23 @@ def subword_reprs(self):
         # Convert strings to a byte tensor.
         #
         # Shape: [batch_size, seq_len, subword_len]
-        subword_bytes = tf.strings.unicode_decode(
-            self.subwords, input_encoding='UTF-8')
-        subword_bytes_padded = subword_bytes.to_tensor(
-            0)[:, :, :self.args.subword_len]
+        with tf.device("/cpu:0"):
+            subword_bytes = tf.cast(self.decode_words(self.subwords, self.args.subword_len), tf.int32)
 
         # Get the lengths of the subwords. Only the last dimension should
         # be ragged, so no actual padding should happen.
```

Comment:
Is the comment about the padding still up to date?

Reply:
Removed.